Dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG RUNC_VERSION=9f9c96235cc97674e935002fc3d78361b696a69e
+
+FROM golang:1.10-alpine AS build-env
+
+RUN apk add --no-cache \
+    zeromq-dev \
+    zeromq \
+    gcc \
+    musl-dev
+
+ADD . /go/src/github.com/caicloud/ciao
+WORKDIR /go/src/github.com/caicloud/ciao
+RUN go build github.com/caicloud/ciao/cmd/kubeflow-kernel \
+	&& mv kubeflow-kernel /usr/bin/kubeflow-kernel
+
+FROM golang:1.10-alpine AS gobuild-base
+
+RUN apk add --no-cache \
+	bash \
+	build-base \
+	gcc \
+	git \
+	libseccomp-dev \
+	linux-headers \
+	make
+
+FROM gobuild-base AS runc
+ARG RUNC_VERSION
```

Comment:
do you need to check out to this version?

Reply:
This is copied from https://github.com/genuinetools/img/blob/master/Dockerfile